SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Uyuni follows a rolling release model, so it is always the next upcoming version that will contain bugfixes for the current one as well as any new features. As there will be no maintenance of specific versions, users will always need to update to the most recent version in order to receive the latest security updates.
+
+## Reporting a Vulnerability
+
+Please report any security vulnerabilities by creating a [security advisory](https://github.com/uyuni-project/uyuni/security/advisories/new). We will be reviewing your report and get back to you after evaluation. We would be happy to give you credits for your help with making Uyuni more secure.
```

Comment:
```suggestion
Please report any security vulnerabilities by creating a [security advisory](https://github.com/uyuni-project/uyuni/security/advisories/new). We will be reviewing your report and get back to you after evaluation. We will be happy to give you credits for your help with making Uyuni more secure.
```

Reply:
Thanks, included this suggestion in a `fixup!` commit.